src/js/utils/NavigationServiceUtil.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import NavigationService from '../../../foundation-ui/navigation';
+
+const NavigationServiceUtil = {
+  /**
+   * routesToDefinition - declares Navigation Elements based on routes using
+   * NavigationService API
+   *
+   * @param  {Array} routes Application routes
+   * @return {undefined}
+   */
+  routesToDefinition(routes) {
```

Comment:
Please rename this util so that it's clear that it actually registers categories and links on the navigation service